### ISSUE TITLE
about GPU memory size

### ISSUE THREAD

**Issue by universefall** [IlchaeJung/RT-MDNet#21]
thanks for your work，my gpu is 2GB. but still shows out of memory. And I try to run it on CPU and change the GPU setting to FALSE. However, it cannot work.

**Comment by IlchaeJung** [comment]
Thank you for your interest in my work.
Unfortunately, the CPU version is not available currently. 
I just make a CPU mode to implement in the future. 
Actually, there is some trick to run this code by reducing the batch_size.
Please could you try this?

**Comment by universefall** [comment]
Yes, I tried to decrease the 'batch_test' from 256 to 2.However, still shows  CUDA error: out of memory. My GPU is GTX1050.
